src/components/views/rooms/RoomHeader.js

```diff
@@ -225,12 +226,17 @@ module.exports = React.createClass({
                 roomName = this.props.room.name;
             }
 
+            const innerName =
+                <EmojiText element="div"
+                           className={ "mx_RoomHeader_nametext " + (settingsHint ? "mx_RoomHeader_settingsHint" : "") }
```

Comment:
It'd be _really_ awesome if this used `classnames` which can be imported from `/src`. Then the indentation could be a bit nicer :grinning: (i.e. `someClasses = classnames({...})` ... `className={someClasses}` )